src/components v2/Atoms/Forms/Label/Label.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div class="photon-label" :class="[size]">
+    <span class="label">{{ label }}</span>
+    <span v-if="isRequired" class="required-asterisks">*</span>
+    <span class="inline-text">{{ inlineText }}</span>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+
+export default Vue.extend({
+  name: 'P2Label',
+  props: {
+    label: {
+      type: String as PropType<string>,
+      default: '',
+    },
+    isRequired: {
+      type: Boolean as PropType<boolean>,
+      default: false,
+    },
+    inlineText: {
+      type: String as PropType<string>,
+      default: '',
+    },
+    size: {
+      type: String as PropType<string>,
+      default: 'sm',
+    },
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+.photon-label {
+  display: flex;
+  align-items: baseline;
+  margin-bottom: 10px;
+
+  .label {
+    font-family: var(
+      --form-control-items-label-default-text-font-family,
+      "'roboto' arial sans-serif"
```

Comment:
Seems like this is the only component we default back to a font declaration. Is it intentional or a relic?

Reply:
Relic I should get rid of this. If the theme doesn't load we've done something wrong.

Reply:
Font-family fallbacks are coming in the next version of Figma Tokens so we can build our branded fallback stack in JSON within 2-3 weeks from now.